pennylane/resources/first_quantization.py

```diff
@@ -15,505 +15,559 @@
+    Here, we assume :math:`\epsilon_{\mathcal{M}} = \epsilon_R = \epsilon_T = \alpha \epsilon` with
+    a default value of :math:`\alpha = 0.01` and obtain
 
+    .. math::
+        \epsilon_{qpe} = \sqrt{\epsilon^2 [1 - (3 \alpha)^2]}.
 
-def norm(eta, n, omega, error, br=7, charge=0):
-    r"""Return the 1-norm of a first-quantized Hamiltonian in the plane-wave basis.
-
-    The expressions needed for computing the norm are taken from
-    [`PRX Quantum 2, 040332 (2021) <https://link.aps.org/doi/10.1103/PRXQuantum.2.040332>`_]. The
-    norm is computed assuming that amplitude ampliﬁcation is performed.
+    Note that the user only needs to define the target algorithm error :math:`\epsilon`. The error
+    distribution takes place inside the functions.
 
     Args:
-        eta (int): number of electrons
         n (int): number of basis states
+        eta (int): number of electrons
         omega (float): unit cell volume
         error (float): target error in the algorithm
-        br (int): number of bits for ancilla qubit rotation
         charge (int): total electric charge of the system
-
-    Returns:
-        float: 1-norm of a first-quantized Hamiltonian in the plane-wave basis
-
-    **Example**
-
-    >>> eta = 156
-    >>> n = 10000
-    >>> omega = 1145.166
-    >>> error = 0.001
-    >>> norm(eta, n, omega, error)
-    1254385.059691027
+        br (int): number of bits for ancilla qubit rotation
 
     .. details::
-        :title: Theory
-
-        For numerical convenience, we use the following modified expressions for computing
-        parameters that contain a sum over the elements, :math:`\nu`, of the set of reciprocal
-        lattice vectors, :math:`G_0`.
-
-        For :math:`\lambda_{\nu}` defined in Eq. (25) of
-        `PRX Quantum 2, 040332 (2021) <https://link.aps.org/doi/10.1103/PRXQuantum.2.040332>`_ as
-
-        .. math::
+        :title: Usage Details
 
-            \lambda_{\nu} = \sum_{\nu \in G_0} \frac{1}{\left \| \nu \right \|^2},
+        .. code-block:: python
 
-        we follow Eq. (F6) of
-        `PRX 8, 011044 (2018) <https://journals.aps.org/prx/abstract/10.1103/PhysRevX.8.011044>`_
-        and use
+            import pennylane as qml
+            from pennylane import numpy as np
 
-        .. math::
+            n = 10000
+            eta = 156
+            omega = 1145.166
 
-            \lambda_{\nu} = 4\pi \left ( \frac{\sqrt{3}}{2} N^{1/3} - 1 \right) + 3 - \frac{3}{N^{1/3}}
-            + 3 \int_{x=1}^{N^{1/3}} \int_{y=1}^{N^{1/3}} \frac{1}{x^2 + y^2} dydx.
+            algo = DoubleFactorization(n, eta, omega)
 
-        We also need to compute :math:`\lambda^{\alpha}_{\nu}` defined in Eq. (123) of
-        `PRX Quantum 2, 040332 (2021) <https://link.aps.org/doi/10.1103/PRXQuantum.2.040332>`_
+        >>> algo.lamb  # the 1-Norm of the Hamiltonian
+        281053.7561435654
 
-        .. math::
+        >>> algo.gates  # estimated number of non-Clifford gates
+        3386558458840
```

Comment:
It's always a bit hard to read large numbers like this. We can either use something like
```python
print(f"{algo.gates}:.2e")
```
which should print the number in scientific notation with 2 decimal precision, or simply do it ourselves directly with 
```
>>> algo.gates
3.39e12